### PR TITLE
Initial charter draft for Community Structure WG

### PR DESCRIPTION
**Pull request by eddie-knight**
Collaboration with @sshiells-scottlogic

### CODE REVIEW

File: docs/governance/working-groups/community-structure/charter.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+- Interface with FINOS team to implement repository controls
+- Propose and maintain community policies and guidelines
+- Provide advice and support to WGs
+
+### Output / Deliverables
+
+This group should, at minimum, create and maintain the following items in the project GitHub repository.
+
+- Community Group Strategy, including:
```
Comment (sshiells-scottlogic) [inline docs/governance/working-groups/community-structure/charter.md:29]: Is there a difference betwen "Community Groups" and "Working Groups"?

File: docs/governance/working-groups/community-structure/charter.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Community Cadence
+
+A member of this working group should be present on for any calls scheduled by the SteerCo for participation by the full CCC community, such as quarterly SteerCo calls.
+
+This working group will use the mail group <ccc-structure@lists.finos.org> for regular communications.
+
+## Changes
+
+Committee members may propose a change to this document through the following process:
```
Comment (sshiells-scottlogic) [inline docs/governance/working-groups/community-structure/charter.md:58]: Who/what are committee members? SteerCo?

**Review by sshiells-scottlogic — COMMENTED** [review]
Looks good, I have left some comments.  There are a couple of questions around definitions of terms used throughout the doc and some small comments on styling/formatting. Nothing major.

File: docs/governance/working-groups/community-structure/charter.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Community Structure Working Group
+
+This working group facilitates the structure and governance design for the Common Cloud Controls project community.
```
[the remark this answers is not in the document]
Comment (eddie-knight) [reply docs/governance/working-groups/community-structure/charter.md:3]: good shout.

note for us to remember later... we should have WG defined in the community groups file and then we can just put a linked [WG] in all of the charters in the same way that I started doing with [SC]

**Comment by sshiells-scottlogic** [comment]
The changes look good to me.

**Review by sshiells-scottlogic — APPROVED** [review]
Changes look good to me.

**Comment by pgste** [comment]
Looks good, I like having someone review the way we work. It will be important to have a regular feedback lop for this.

**Review by AdrianHammond — COMMENTED** [review]
+1 for this  - looks good

**Review by rgriffiths-scottlogic — APPROVED** [review]
Review and Approved

**Pull request merged**